data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/model/EnvironmentVersion.java

```diff
@@ -19,11 +19,10 @@
 
 @Value
 @Builder
-public class EnvironmentRevision {
-  private final String environmentArn;
-  // TODO: Maybe this should just be an environmentRevisionArn, instead of a separate arn and version?
+public class EnvironmentVersion {
+  private final String environmentId;
   private final String environmentVersion;
   private final String taskDefinition;
-  private final DeploymentMethod deploymentMethod;
+  private final DeploymentType deploymentType;
```

Comment:
why do we need this here?

Reply:
So that we can determine which scheduling algorithm to use once we reach the `Scheduler` function: https://github.com/blox/blox/pull/272/files#diff-0e6557774ed72aae34818e81eafbb6ebR19